src/grisp_connect_updater_progress.erl

```diff
@@ -41,10 +41,16 @@ progress_init(#{client := PID} = _Opts) ->
 
 progress_update(#state{last_notification = LastLog} = State, Stats) ->
     case (erlang:system_time(millisecond) - LastLog) > 1000 of
-        false -> {ok, State};
-        true ->
+        false -> 
+            {ok, State};
+        true  -> 
+            UpdatePercentage = progress_percent(Stats),
             % Recheck log level when there is another way to check the progress update
-            ?LOG_NOTICE("Update progress: ~b%", [progress_percent(Stats)]),
+            ?LOG_NOTICE("Update progress: ~b%", [UpdatePercentage]),
+            grisp_connect_client:request(
+                <<"notify">>,
+                <<"status_update">>, 
+                #{percentage => UpdatePercentage}),
```

Comment:
@IciaCarroBarallobre you should not use a jsonrpc request but a notification , see https://www.jsonrpc.org/specification.

You basically need to add functions like `grisp_connect_client:notify` and `grisp_connect_api:notify` to handle jsonrpc notifications. They are similar to requests, they just do not need the ID since there is no expected reply.

Reply:
I'm working on it